### ISSUE TITLE
`TypeError: marker is null` when returning empty strings

### ISSUE THREAD

**Issue by bluenote10** [ryansolid/solid#32]
The following example throws a `TypeError: marker is null` on the second click on the button:

```js
const App = () => {
  const [state, setState] = createState({enabled: false})
  return (
    <div>
      <div>Enabled: {(state.enabled ? "yes" : "")}</div>
      <button onclick={(event) => setState({enabled: !state.enabled})}>Toggle</button>
    </div>
  )
};
```

The problem seems to be caused by using an empty string literal `""` in the `state.enabled === false` case, because changing it to a non-empty literal works. Interestingly it also works on the initial rendering, just not on a re-render.

I should probably use `<$>` for conditional rendering in general, but I would have expected that the empty string is still handled correctly.

**Comment by ryansolid** [comment]
For something that simple ternary operator is good. You only want the overhead of control flow when dealing with nested nodes/computed contexts. I have been generalizing the code between control flow and normal inserts the last couple releases and I probably am missing a couple cases in the tests. I will look at this, this evening and get you fix.

**Comment by ryansolid** [comment]
Yeah what I thought. Just introduced it yesterday. Missed a null check in one place. I've added new tests to dom-expressions. It is fixed in 0.7.5.

**Comment by bluenote10** [comment]
Great, thanks for the quick fix!
